frontend/src/bundles/studio/components/timeline/timecursor/timecursor.tsx

```diff
@@ -0,0 +1,137 @@
+import { useTimelineContext } from 'dnd-timeline';
+
+import { Box } from '~/bundles/common/components/components.js';
+import {
+    useCallback,
+    useLayoutEffect,
+    useRef as useReference,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+
+type Properties = {
+    interval?: number;
+};
+
+const now = Date.now();
+
+const TimeCursor: React.FC<Properties> = ({
+    interval,
+}: Properties): JSX.Element => {
+    const timeCursorReference = useReference<HTMLDivElement>(null);
+    const renderTimeReference = useReference(now);
+    const { range, direction, sidebarWidth, valueToPixels, pixelsToValue } =
+        useTimelineContext();
+
+    const side = direction === 'rtl' ? 'right' : 'left';
+
+    const [isDragging, setIsDragging] = useState(false);
+    const [cursorPosition, setCursorPosition] = useState<number | null>(null);
+
+    useLayoutEffect(() => {
+        const offsetCursor = (): void => {
+            if (!timeCursorReference.current || cursorPosition !== null) {
+                return;
+            }
+            const timeDelta = now - renderTimeReference.current;
+            const timeDeltaInPixels = valueToPixels(timeDelta);
+
+            const sideDelta = sidebarWidth + timeDeltaInPixels;
+            timeCursorReference.current.style[side] = `${sideDelta}px`;
+        };
+        offsetCursor();
+        const cursorUpdateInterval = setInterval(
+            offsetCursor,
+            interval ?? 1000,
```

Comment:
same here, we can you use 1000 for the same purpose

Reply:
Fixed